lib/matplotlib/dviread.py

```diff
@@ -882,6 +996,27 @@ def get_metrics(self, idx):
         property(lambda self: {c: m.tex_depth for c, m in self._glyph_metrics}))
 
 
+class TtfMetrics:
+    def __init__(self, filename):
+        self._face = font_manager.get_font(filename, hinting_factor=1)
+
+    def get_metrics(self, idx):
+        # _mul1220 uses a truncating bitshift for compatibility with dvitype.
+        # When upem is 2048 the conversion is exact to 12.20 is exact, but when
```

Comment:
"is exact" appears twice in this sentence.

Reply:
fixed